commit 152ca50ed9cbee9222931324bea6d6490719a428
Author: Friedrich W. H. Kossebau <kossebau@kde.org>
Date:   Mon Sep 19 16:35:31 2022 +0200

    Update homepage to apps.kde.org
    
    GIT_SILENT
    
    (cherry picked from commit 37d732d20d9df9a9511a4ddfa1574645f5b13ccc)

diff --git a/src/main.cpp b/src/main.cpp
index 0b1976f..bf066a0 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -43,7 +43,7 @@ int main(int argc, char **argv)
             KUBRICK_VERSION_STRING, i18n ("A game based on Rubik's Cube (TM)"),
             KAboutLicense::GPL,
             i18n ("&copy; 2008 Ian Wadham"),
-            QStringLiteral("https://kde.org/applications/games/org.kde.kubrick") );
+            QStringLiteral("https://apps.kde.org/kubrick") );
     about.addAuthor (i18n ("Ian Wadham"), i18n ("Author"),
                              QStringLiteral("iandw.au@gmail.com"));
 #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
diff --git a/src/org.kde.kubrick.appdata.xml b/src/org.kde.kubrick.appdata.xml
index 152b443..da71020 100644
--- a/src/org.kde.kubrick.appdata.xml
+++ b/src/org.kde.kubrick.appdata.xml
@@ -112,6 +112,7 @@
     <p xml:lang="uk">Kubrick — це гра, яку засновано на головоломці Кубик Рубіка™. Розміри кубика можуть змінюватися у межах від 2x2x2 до 6x6x6, крім того, ви можете грати з неправильними «цеглинками», з розмірами на зразок 5x3x2, або «килимками», з розмірами на зразок 6x4x1 або 2x2x1. У цій грі ви можете обирати головоломки декількох рівнів складності, спостерігати за демонстрацією чудових візерунків та обертань розв’язання та створювати власні головоломки.</p>
     <p xml:lang="x-test">xxKubrick is a game based on the Rubik's Cube™ puzzle. The cube sizes range from 2x2x2 up to 6x6x6, or you can play with irregular "bricks" such as 5x3x2 or "mats" such as 6x4x1 or 2x2x1. The game has a selection of puzzles at several levels of difficulty, as well as demos of pretty patterns and solution moves, or you can make up your own puzzles.xx</p>
   </description>
+  <url type="homepage">https://apps.kde.org/kubrick</url>
   <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?format=guided&amp;product=kubrick</url>
   <url type="help">https://docs.kde.org/?application=kubrick</url>
   <url type="donation">https://www.kde.org/community/donations/?app=kubrick&amp;source=appdata</url>
